crawler/googleCrawler.py

```diff
@@ -0,0 +1,72 @@
+import googlemaps
+import json
+import requests
+import os
+import time
+
+from bs4 import BeautifulSoup
+from os.path import join, dirname
+from dotenv import load_dotenv, find_dotenv
+
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(find_dotenv(), override=True)
+
+API_KEY = os.environ.get("API_KEY")
+gmaps = googlemaps.Client(key=API_KEY)
+
+fread = open('zipcode.txt', 'r')
+while True:
+    line = fread.readline()
+    if not line: break
+    zipcode = line[0:5]
+
+    query_str = "San,Francisco,CA," + zipcode
+    language_str = "en"
+    type_str = "cafe"
+    page_token = ""
+
+    result_list = []
+    for idx in range(3):
+        time.sleep(8)
```

Comment:
Is this to limit the qps rate limit? How did you arrive at the value 8? Leave the rationale as a comment above this line.